pkg/jx/cmd/get_applications.go

```diff
@@ -151,6 +154,19 @@ func (o *GetApplicationsOptions) generateTable(apps []string, envApps []EnvApps,
 	appEnvMap := map[string]map[string]*ApplicationEnvironmentInfo{}
 	for _, appName := range apps {
 		row := []string{appName}
+		jxClient, ns, err := o.JXClientAndDevNamespace()
+		if err != nil {
+			log.Warnf("Error getting jx client and dev namespace: %s", err)
+		}
+		sourceRepositories := jxClient.JenkinsV1().SourceRepositories(ns)
+		sourceRepository, err := GetSourceRepositoryForApplication(appName, sourceRepositories)
+		gitURL := "None Found"
+		if err != nil {
+			log.Warnf("Could not get Source Repository for application: %s", err)
```

Comment:
Do we need to log here a warning or just go silent with no git URL? Maybe this will confuse the user.

Reply:
That's a good question. I don't have strong feelings about this, it just felt right to add an error message here.